apps/web/src/pages/details/[applicantId].tsx

```diff
@@ -82,11 +82,7 @@ const Details = () => {
         <div className='col-span-3'>
           <DetailsItem
             title='Assigned To'
-            text={
-              applicant.assigned_to
-                ? applicant.assigned_to.map((a: { name: any }) => a.name).join(',')
-                : 'NA'
-            }
+            text={applicant.assigned_to?.map((a: { name: any }) => a.name).join(',') || 'NA'}
```

Comment:
can we replace 'any' with 'string' here(' name: any')?

Reply:
both those were fixed in my most recent PR

Reply:
@bhumin-fw 
`nursing_educations`, `health_authorities`, and `assigned_to` seem to be an array, so how about changing their type 
from JSON to array of their type?

Reply:
@sunghwan-park-fw That's a good idea. We can put an array of objects. You can update it in this PR, or I can push the same in my next PR whatever is suitable for you.

Reply:
Please, update RO then I will update components referencing the fields.